solidus_shipstation.gemspec

```diff
@@ -31,4 +31,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'sass-rails'
   s.add_development_dependency 'sqlite3'
   s.add_development_dependency 'rspec-xsd'
+  s.add_development_dependency 'pry'
+  s.add_development_dependency 'pry-byebug'
```

Comment:
I will remove this for now, I think we need to default to pry or to irbtools. But nice gem 👍